source/Camera/Basic/CaptureAssistant/CaptureAssistant.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-This example shows how to use Capture Assistant to capture point clouds, with color, from the Zivid camera.
```

Comment:
Remember to change this in other sample repos when you start 
![image](https://user-images.githubusercontent.com/46567886/87550944-ee6e3d80-c6af-11ea-85c1-dbc011bb5e49.png)

If you think this is important it should be modified in all samples that have it and also in the README.md. Then it should be also modified in the other repos. I personally think it would be ok to skip it. I think both versions are gramatically correct.

Reply:
I think you're right. We can just leave the comma in.  It is just a personal style choice.